src/views/Puzzle/Puzzle.tsx

```diff
@@ -779,6 +779,7 @@ export class Puzzle extends React.Component<PuzzleProperties, PuzzleState> {
             <div className="btn-container">
                 <div className="btn-group">
                     <button type="button"
+                        title="Flip Diagonal"
```

Comment:
What I meant in the other comment:

```title={_("Flip diagonal")}```

Alternatively, if you want to add some context:

```title={pgettext("Puzzle options", "Flip Diagonal")```

The translators will be able to see this in their dashboard and add a translation later.

Reply:
Fixed!

Reply:
Cool, thanks!